js/src/components/paid-ads/asset-group/asset-group-section.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import './asset-group-section.scss';
+
+/**
+ * Renders the form content for managing an asset group of a campaign with Section UI.
```

Comment:
💅 

Feels for me that the `form content for managing asset group of a campaign` will be handed in  another (form?) component (where now is a final url placeholder).

I see this more like a `Renders the section UI for managing asset group of a campaign`

Reply:
This component is at the same hierarchy level as several components in the codebase named `*FormContent`. However, the form "content" is usually split into several components and combined with some other non-form components at different hierarchy levels to composite the final form content, so I'm not keen on having a specific component named `FooFormContent`. The "form content" is used here in place of the `FormContent` part that was originally included in its naming.